text_mining.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import re
+import string
+from bs4 import BeautifulSoup
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+#Samantha Young
+
+# Getting Harry Potter 1 from archive .org
+
+html1 = BeautifulSoup(requests.get("https://archive.org/stream/Book5TheOrderOfThePhoenix/Book%201%20-%20The%20Philosopher's%20Stone_djvu.txt").text, 'lxml')
+html1.find('pre')  # find the story
+str(html1.find('pre'))  # the story, as a string. Includes embedded <b> etc.
+textfromsite1 = str(re.sub(r'<.+?>', '', str(html1.find('pre')))) #saving the story as a string
+
+# Getting Harry Potter 5 from archive .org
+
+html5 = BeautifulSoup(requests.get("https://archive.org/stream/Book5TheOrderOfThePhoenix/Book%205%20-%20The%20Order%20of%20the%20Phoenix_djvu.txt").text, 'lxml')
+html5.find('pre')  # find the first paragraph
+str(html5.find('pre'))
+textfromsite5 = str(re.sub(r'<.+?>', '', str(html5.find('pre'))))
+
+#H Getting Harry Potter 7 from archive .org
+
+html7 = BeautifulSoup(requests.get('https://archive.org/stream/Book5TheOrderOfThePhoenix/Book%207%20-%20The%20Deathly%20Hallows_djvu.txt').text, 'lxml')
+html7.find('pre')  # find the first story
+str(html7.find('pre'))
+textfromsite7 = str(re.sub(r'<.+?>', '', str(html7.find('pre')))) #saving the story as a string
+
```

Comment:
I would suggest separating your data-collection part of this file and the text analysis part into two separate files. In this way, you don't have to make a data request every time you run the analysis.